client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -270,14 +287,22 @@ const fixedProductModel = {
 		labelText: translate( 'Coupon Code & Discount' ),
 		cssClass: 'promotions__promotion-form-card-primary',
 		fields: {
-			couponCode: couponCodeField,
-			fixedDiscount: (
-				<CurrencyField
-					labelText={ translate( 'Product Discount', { context: 'noun' } ) }
-					isRequired
-				/>
-			),
-			freeShipping: freeShippingField,
+			couponCode: {
+				component: couponCodeField,
+				validate: validateCouponCode,
+			},
+			fixedDiscount: {
+				component: (
+					<CurrencyField
+						labelText={ translate( 'Product Discount', { context: 'noun' } ) }
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.